internal/linter/testdata/validcfg/.skeema

```diff
@@ -10,4 +10,6 @@ ignore-table=^_
 
 schema=whatever
 default-character-set=latin1
-default-collation=latin1_swedish_ci
\ No newline at end of file
+default-collation=latin1_swedish_ci
+
+allow-pk-type=int,bigint,varbinary
```

Comment:
In autoinc.sql, table `smallautoinc` is using a smallint column for its PK, so it's getting flagged for pk-type. Could fix by adding smallint to this list in the .skeema file, or by adding pk-type to the list of expected annotations on line 14 of autoinc.sql, either way is fine. Thanks!